### DISCUSSION TITLE
Purge Line Issue

### DISCUSSION THREAD

**Discussion by cadbob** [Guilouz/Creality-Helper-Script-Wiki#463] [Issues]
I Reset older firmware helper script, doing factory reset and re-rooted with Newest Script. My Purge Line kept coming in almost exactly at edge of Model Being printed. So I unchecked Exclude Object now get the Classic purge line. I would like to have Exclude Object checked in case of multiple parts but can't as most prints glob up at purge line. looking for advise.

**Comment by cadbob** [comment]
Update:   Flashed new Firmware 1.3.3.8 and re-rooted the Purge Line Seems to be out of Printing Path Now thou still close, I just tested with Calibration Cube so it doesnt have a brim which purge line was into on my other prints. Fingers Crossed that re-rooting fixed issue. Wish we could control Purge Line location, or if we can please someone let me in on instructions for doing it.
   My thoughts are the Program is not taking into account the Brim Support when figuring where to lay down purge line it only looks to the actual Model that was sliced. Could be wrong but my best guess

**Reply by OriginalHotrodder** [reply]
I think you are right on the money with the purge line not taking into account the Brim Support. I'm printing an object which has an auto brim of about 10mm and the purge line is half way into the brim.

**Comment by JooJooBee666** [comment]
I'm having the same issue.  Is there ANY way at all to move this say...to the total front of the plate?  It's constantly getting in the way for me with my tree supports and it screws up the entire print.  I have to watch and WHILE it's printing and moving, stick my hand in there and pull it off before anything prints over it and screws up the nozzle (it starts dropping blobs of crap every where and causing the print to lift or deform.

How do we change this?
